cluster/engine.go

```diff
@@ -935,7 +935,7 @@ func (e *Engine) UpdateNetworkContainers(containerID string, full bool) error {
 	containerMap := make(map[string]*Container)
 	if containerID != "" {
 		ctr, err := e.refreshContainer(containerID, full)
-		if err != nil {
+		if err != nil || ctr == nil {
```

Comment:
Actually I am not sure I like combine `err != nil` and `ctr == nil` with `||`.
I prefer split them into two blocks, like:
```
if err != nil {
        return err
}

if ctr == nil {
        return ...
}
```

WDYT?